### PR TITLE
Updates to the happy path readme(s)

### PR DESCRIPTION
**Pull request by jjasghar**
* Adam and I spent the 4th cd.foudation twitch stream making edits to
the readme(s) for a happy path deployment.

- Moved the getting-started to a more prominent location
- Added a note about how to get the ingress
subdomain/external IP for the webhook
- Made a note of secret name from above step, and
export the secreat to a yaml then edit namespace line to namespace and
import secreat back
- Added the value of "ExternalDomain" as a "minimal update"

Signed-off-by: JJ Asghar <jjasghar@gmail.com>
Co-authored-by: Adam Roberts <aroberts@uk.ibm.com>

### CODE REVIEW

File: docs/getting-started/README.md [hunk 1]
```diff
@@ -34,11 +34,23 @@ Now that we have our cluster ready, we need to setup our getting-started
 namespace and RBAC. We will keep everything inside this single namespace for
 easy cleanup. In the unlikely event that you get stuck/flummoxed, the best
 course of action might be to just delete this namespace and start fresh.
+You should take not of the _ingress_ subdomain, or the external IP of
```
Comment (a-roberts) [inline docs/getting-started/README.md:37]: nit: note

File: docs/getting-started/README.md [hunk 2]
```diff
@@ -34,11 +34,23 @@ Now that we have our cluster ready, we need to setup our getting-started
...
+your cluster, you will need this for your Webhook in later steps.
 
 - Create the _getting-started_ namespace, where all our resources will live.
   - `kubectl create namespace getting-started`
 - [Create the admin user, role and rolebinding](./rbac/admin-role.yaml)
   - `kubectl apply -f ./docs/getting-started/rbac/admin-role.yaml`
+- If you have a cluster secret for a Let's Encrypt cert already provisisoned
```
Comment (a-roberts) [inline docs/getting-started/README.md:44]: nit: provisioned

File: docs/getting-started/README.md [hunk 3]
```diff
@@ -34,11 +34,23 @@ Now that we have our cluster ready, we need to setup our getting-started
...
+you will need to export it, and reimport it to the _getting-started_ namespace.
+The following is a general example of what you'd need to do.
+  - ```bash
```
Comment (a-roberts) [inline docs/getting-started/README.md:47]: Looks like

```
kubectl get secret <name> --namespace=<namespace> --export -o yaml |\
   kubectl apply --namespace=<new namespace> -f -
```

would work too, taken from https://www.revsys.com/tidbits/copying-kubernetes-secrets-between-namespaces/

File: docs/getting-started/README.md [hunk 4]
```diff
@@ -157,6 +174,7 @@ minimum.
 - GitHubOrg: The GitHub org you are using for this getting-started.
 - GitHubUser: Your GitHub username.
 - GitHubRepo: The repo we will be using for this example.
+- ExternalDomain: Update this to the to something other then `demo.iancoffey.com`
```
Comment (a-roberts) [inline docs/getting-started/README.md:177]: nit: update this to be*

**Review by a-roberts — CHANGES_REQUESTED** [review]
Added my comments - mainly just nits and what could be a slight enhancement for the secret copying

Thanks for working on this @jjasghar! All for happy paths 😄

**Comment by dibyom** [comment]
Thanks @jjasghar @a-roberts 

Once the nits are addressed, this is good to go!

**Review by savitaashture — COMMENTED** [review]
/lgtm

**Comment by savitaashture** [comment]
Can you squash the commits into one?

**Pull request merged**